index.html

```diff
@@ -2642,8 +2642,9 @@ <h3>
         </h3>
         <p>
           The <dfn>url</dfn> member of a <a>ShortcutItem</a> is the <a>URL</a>
-          <a data-lt="within scope of a manifest">within the application's
-          scope</a> that opens when the associated shortcut is activated.
+          that opens when the associated shortcut is activated. When resolved,
+          the {{ShortcutItem/url}} must be [=URL/within scope=] of scope URL,
+          it gets ignored.
```

Comment:
This sentence is ungrammatical ("it gets ignored"). Also, do not use the keyword "must" in this definition which is not strictly imposing requirements. Just say "is the URL [=manifest/within scope=] of the manifest that opens when the associated shortcut is activated." (It doesn't have to be explicit about everything, since that's covered by algorithms elsewhere.)

Reply:
agree... will drop the redundant part.